src/main/java/utils/FormBuilder.java

```diff
@@ -97,13 +100,33 @@ public FormBuilder addTextField(String name, String display, String placeholder,
         return this;
     }
 
-    public FormBuilder addTextArea(String name, String display, String placeholder, boolean required) {
-        String textAreaML = TagBuilder.builder("textarea")
+    public FormBuilder addTextField(String name, String display, String placeholder, boolean required, boolean masked) {
+        TagBuilder textFieldML = TagBuilder.builder("text-field")
```

Comment:
Are you sure that modifying this method prototype doesn't break backward compatibility with previous versions? Reminder: we have to be very careful with SDKs public methods that might be already in use by some of our customers. 

If there is a possibility for this method to be use, `@Deprecated` it and create another one with the additional parameter `boolean masked`

Reply:
@symphony-thibault are we good with these changes?